Makefile

```diff
@@ -8,6 +8,11 @@ build:
 	mkdir -p ~/.terraform.d/plugins/vmware/dev/tanzu-mission-control/1.0.0/darwin_amd64/
 	cp bin/terraform-provider-tanzu-mission-control_v1.0.0 ~/.terraform.d/plugins/vmware/dev/tanzu-mission-control/1.0.0/darwin_amd64/
 
+build-arm:
```

Comment:
It's same as make build?

Reply:
Yes, by default the build will build for amd (which is still the majority). The new build tag is for arm, which is the architecture used by Mac OS

Reply:
There should be some difference in the path used for arm?

Reply:
amd and arm are different